### ISSUE TITLE
Prevent method from a 2nd slider to be called

### ISSUE THREAD

**Issue by heyjudemaldicas** [vueform/slider#52]
I have 2 sliders linked by certain calculations. The 2nd slider's v-model `downpayment.value` is being updated whenever the 1st slider is dragged `selectPrice()`. This triggers the update for method `selectDownpayment()`, which should not be the case. Is there anything that I can do to not trigger it?

https://codesandbox.io/s/focused-andras-u9xv20

**Comment by heyjudemaldicas** [comment]
I just realized I posted a wrong assumption here after debugging again (yup, my bad).

Using below image as an example, an initial price 45000 gives me a downpayment value of 14445. Updating the price from 45000 to 44500 should give me a downpayment value of 14285. But checking the logs, it returned the old value which is 14445.

![drag](https://user-images.githubusercontent.com/41458924/161770269-a6c469f9-242c-48e7-b0ff-8f1c344c87b9.jpg)

I realized that updating `downpayment.value` in my `selectPrice` method is "NOT REALLY" updating the slider value. And when it moves to `selectDownpayment`, the old value of `downpayment.value` is the one being returned.

So my question is, is there a way to force update `downpayment.value` immediately?
